test/px-data-table-fixture.html

```diff
@@ -174,5 +174,30 @@
         </px-data-table>
       </div>
 
+      <p>Smart Row Selection, client</p>
+      <div id="styledContainer2" style=" height:800px; border: 1px solid cyan; flex:1; display:flex; flex-direction:column; ">
+        <px-data-table
+          enable-column-resize="true"
+          greedy-height-with-scroll="true"
+          id="clientSmartRowSelectionWithLargeHeight"
+          show-column-chooser="true"
+          table-rows="true"
+          >
+        </px-data-table>
+      </div>
+
+      <p>Smart Row Selection, server</p>
+      <div id="styledContainer3" style=" height:900px; border: 1px solid cyan; flex:1; display:flex; flex-direction:column; ">
```

Comment:
`styledContainer` could be more descriptive. Perhaps: `flexContainer`, or `fixedHeightFlexContainer`?